frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1107,6 +1112,45 @@ function ContextMenuInner(propsWithInputRef: Props) {
     maybeViewport,
   } = props;
 
+  const segmentIdAtPosition = globalPosition != null ? getSegmentIdForPosition(globalPosition) : 0;
+  const { visibleSegmentationLayer, volumeTracing } = props;
+  const hasNoFallbackLayer =
+    visibleSegmentationLayer != null &&
+    "fallbackLayer" in visibleSegmentationLayer &&
+    visibleSegmentationLayer.fallbackLayer == null;
+  const [segmentVolume, boundingBoxInfo] = useFetch(
+    async () => {
+      if (contextMenuPosition == null) return [];
+      if (visibleSegmentationLayer != null && volumeTracing != null && hasNoFallbackLayer) {
+        const tracingId = volumeTracing.tracingId;
+        const tracingStoreUrl = Store.getState().tracing.tracingStore.url;
+        const mag = getResolutionInfo(visibleSegmentationLayer.resolutions);
+        const [segmentSize] = await getSegmentVolumes(
+          tracingStoreUrl,
+          tracingId,
+          mag.getHighestResolution(),
+          [segmentIdAtPosition],
+        );
+        const [boundingBox] = await getSegmentBoundingBoxes(
+          tracingStoreUrl,
+          tracingId,
+          mag.getHighestResolution(),
+          [segmentIdAtPosition],
+        );
+        const boundingBoxTopLeftString = `(${boundingBox.topLeft[0]}, ${boundingBox.topLeft[1]}, ${boundingBox.topLeft[2]})`;
+        const boundingBoxSizeString = `(${boundingBox.width}, ${boundingBox.height}, ${boundingBox.depth})`;
+        return [
+          formatNumberToVolume(segmentSize),
+          `${boundingBoxTopLeftString}, ${boundingBoxSizeString}`,
+        ];
+      }
+    },
+    ["loading", "loading"],
+    // Update segment infos when opening the context menu, in case the annotation was saved since the context menu was last opened.
+    // Of course the info should also be updated when the menu is opened for another segment, or after the refresh button was pressed.
+    [contextMenuPosition, segmentIdAtPosition, lastTimeSegmentInfoWasFetched],
+  ) as [string, string];
```

Comment:
the function doesn't always return a string-tuple. if you fix that, you shouldn't need the casting.